### PR TITLE
Feature#11/create the footer layout

### PR DESCRIPTION
**Pull request by Javi1499**
# Description
What did you implemented
 - Create Footer section using responsive design 
 - Add sections about us, navigation, contact and social into footer.
 - Style was implemented with sass using Flex and grid
 
Why did you implemented this:
 - This changes are focus on show relevant informatio to user and also responsive design was implemented to show views in differents devices.
 
  ## Screenshots 

Captura de pantalla 2022-04-19 - 17 48 24
imagen

### CODE REVIEW

**Review by EliasUR — DISMISSED** [review]
lgtm

**Review by germanruzca — APPROVED** [review]
LGTM

File: app/views/layouts/_footer.html.erb [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+      </div>
+  </section>
+  <div class="footer-copyrigth">
+    <p class="footer-copyrigth__text">Todos los derechos reservados © NOMBREAPP </p>
+  </div>
+  <div class="footer__box">
+    <%=image_tag("Master.png",class:"footer__image", alt: "Master logo") %>
+  </div>
+
+</footer>
```
Comment (AlejandroCuriel) [inline app/views/layouts/_footer.html.erb:46]: Add a line break

File: app/assets/stylesheets/utils/_variables.scss [hunk 2]
```diff
@@ -1,5 +1,10 @@
...
-
+$yellowFF: #FFCD1C;
+$yellowFAF: #FAFF00;
+$graye1: #E1E1E1;
 // Font Family
 
 // Text sizes
+
+//ViewPorts
+$mobile: 700px
```
Comment (AlejandroCuriel) [inline app/assets/stylesheets/utils/_variables.scss:10]: Add a line break

File: app/assets/stylesheets/utils/_variables.scss [hunk 3]
```diff
@@ -1,5 +1,10 @@
 // Colors
-
+$yellowFF: #FFCD1C;
+$yellowFAF: #FAFF00;
+$graye1: #E1E1E1;
 // Font Family
 
 // Text sizes
+
+//ViewPorts
```
Comment (AlejandroCuriel) [inline app/assets/stylesheets/utils/_variables.scss:9]: space between the lines

File: app/assets/stylesheets/layout/_footer.scss [hunk 4]
```diff
@@ -0,0 +1,223 @@
...
+
+  }
+
+  &__box {
+    bottom: 50px;
+    height: fit-content;
+    position: absolute;
+    right: 50px+16px;
+    width: 40px;
+    @media screen and (max-width: $mobile) {
```
Comment (AlejandroCuriel) [inline app/assets/stylesheets/layout/_footer.scss:218]: add empty space before media

File: app/assets/stylesheets/layout/_footer.scss [hunk 5]
```diff
@@ -0,0 +1,223 @@
...
+      color: $yellowFF;
+      cursor: pointer;
+      font-size: 30px;
+      height: 30px;
+      width: 30px;
+    }
+
+    @media screen and (max-width: $mobile) {
+      justify-content: center;
+      &__text {
```
Comment (AlejandroCuriel) [inline app/assets/stylesheets/layout/_footer.scss:182]: add empty space before

File: app/assets/stylesheets/layout/_footer.scss [hunk 6]
```diff
@@ -0,0 +1,223 @@
...
+        display: none;
+      }
+      &__box {
```
Comment (AlejandroCuriel) [inline app/assets/stylesheets/layout/_footer.scss:185]: add empty space before

File: app/assets/stylesheets/layout/_footer.scss [hunk 7]
```diff
@@ -0,0 +1,223 @@
...
+
+      &--title {
+        color: white;
+      }
+    }
+
+    @media screen and (max-width: $mobile) {
+      justify-content: center;
+      text-align: center;
+      &__icon {
```
Comment (AlejandroCuriel) [inline app/assets/stylesheets/layout/_footer.scss:127]: add empty space before

File: app/assets/stylesheets/layout/_footer.scss [hunk 8]
```diff
@@ -0,0 +1,223 @@
...
+        display: none;
+      }
+      &__box {
```
Comment (AlejandroCuriel) [inline app/assets/stylesheets/layout/_footer.scss:130]: add empty space before

File: app/assets/stylesheets/layout/_footer.scss [hunk 9]
```diff
@@ -0,0 +1,223 @@
...
+        justify-content: center;
+        margin: 0 0 16px 0;
```
Comment (AlejandroCuriel) [inline app/assets/stylesheets/layout/_footer.scss:132]: if you only need to modify a specific place in margin, you can use "margin-top", "margin-right" , "margin-bottom", "margin-left"

File: config/routes.rb [hunk 10]
```diff
@@ -1,4 +1,5 @@
 Rails.application.routes.draw do
   devise_for :users
+  get "/", to: "home#index"
```
Comment (AlejandroCuriel) [inline config/routes.rb:3]: if the path is already found as root, is it necessary to add this path?

**Review by AlejandroCuriel — CHANGES_REQUESTED** [review]
Very good job, just need to correct one or another detail

File: app/assets/stylesheets/layout/_footer.scss (hunk 9 above)
Comment (Javi1499) [reply app/assets/stylesheets/layout/_footer.scss:132]: Yes, but in desktop view in the class &__box I put diffent values for margin property and into mobile view I needed modify others side of &__box and delete margin of space of desktop view

**Review by Frcomparan — DISMISSED** [review]
LGTM

**Review by AlejandroCuriel — APPROVED** [review]
LGTM  :rocket: xoxo :100:

**Review by Frcomparan — DISMISSED** [review]
LGTM :thumbsup:

File: app/assets/stylesheets/utils/_variables.scss [hunk 11]
```diff
@@ -1,5 +1,8 @@
 // Colors
-
+$yellowFF: #FFCD1C;
+$yellowFAF: #FAFF00;
+$graye1: #E1E1E1;
```
Comment (YilKanda) [inline app/assets/stylesheets/utils/_variables.scss:4]: Preferably use variable names that are more meaningful, for example: primary, secondary, disabled, enabled, page_background, title, subtitle, etc.

File: app/helpers/home_helper.rb [hunk 12]
```diff
@@ -0,0 +1,2 @@
+module HomeHelper
+end
```
Comment (YilKanda) [inline app/helpers/home_helper.rb:2]: If you don't occupy it, you can delete it.

File: test/controllers/home_controller_test.rb [hunk 13]
```diff
@@ -0,0 +1,7 @@
+require "test_helper"
+
+class HomeControllerTest < ActionDispatch::IntegrationTest
+  # test "the truth" do
+  #   assert true
+  # end
```
Comment (YilKanda) [inline test/controllers/home_controller_test.rb:6]: If you don't occupy it, you can delete it.

**Review by EliasUR — APPROVED** [review]
LGTM

**Review by Frcomparan — APPROVED** [review]

**Pull request merged**